frontend/src/main/scala/bloop/logging/BspServerLogger.scala

```diff
@@ -114,6 +117,23 @@ final class BspServerLogger private (
             event.clear
           )
         )
+      case (_, Some(file)) =>
```

Comment:
How do we know this is coming from `javac` only? It would be great to check that manually and not assume that the position coming from javac will have a very concrete shape.